fastavro/_schema.pyx

```diff
@@ -28,7 +28,7 @@ cpdef inline str extract_logical_type(schema):
     return None
 
 
-def schema_name(schema, parent_ns):
+cpdef schema_name(object schema, parent_ns):
```

Comment:
Was trying to see if it would speed things up, from what I saw it didn't introduce any bugs, but this can be reject or reverted if you're worried about it.

Reply:
I noticed you opened an issue on this, so let's discuss there and potentially make changes in another PR.